redirect/routes.php

```diff
@@ -108,7 +108,7 @@ class Route {
         ],
         'dwt'          => [
             'description' => 'Diskrete Wahrscheinlichkeitstheorie',
-            'target'      => 'http://wwwalbers.in.tum.de/lehre/2020SS/dwt/index.html.de',
+            'target'      => 'https://wwwalbers.in.tum.de/lehre/2020SS/dwt/index.html.de',
```

Comment:
unfortunately this site does not support https :D

Reply:
Oh, I forgot about that change. Thanks